contracts/pre-propose/dao-pre-propose-approver/src/msg.rs

```diff
@@ -10,6 +10,13 @@ pub struct InstantiateMsg {
     pub pre_propose_approval_contract: String,
 }
 
+#[cw_serde]
+pub enum ExecuteExt {
+    // Reset approver back to DAO that set up this approver contract. Only
+    // callable by the DAO.
```

Comment:
Callable by which DAO?

Reply:
This is the approver contract that the approver DAO sets up, which changes the approval's `approver` from the approver DAO to the approver DAO's new approver contract. So to reset, it sets the approval's `approver` back to the approver DAO (which the approval DAO initially set).